fiatconnect-api.md

```diff
@@ -2223,6 +2224,25 @@ platform can be used to consume Fiat Connect services by providing their `mobile
 }
 ```
 
+#### 8.3.2.4. `IfscAccount`
+
+`IfscAccount` is an account schema that represents transfers for `INR` currency.
+
+```
+{
+	accountName: `string`,
+	institutionName: `string`,
+	ifsc: `string`,
+	accountNumber: `string`,
```

Comment:
Is there a general syntax required for `accountNumber` or does it vary from bank to bank? If it's the same syntax for all banks, could you please specify it in the paragraph below?